lineup.js

```diff
@@ -1,9 +1,14 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((total, lineup) => {
```

Comment:
Minor naming point. Since lineup is your original list and you are passing in an individual item from that list, using lineup again for the function you pass to reduce can be a little harder to read. Since a lineup is a collection of players I'd have named this player like you did below.  It still works, just going for readability.